cardano-node/src/Cardano/CLI/Delegation.hs

```diff
@@ -58,15 +61,15 @@ checkByronGenesisDelegation
   -> ProtocolMagicId
   -> Crypto.VerificationKey
   -> Crypto.VerificationKey
-  -> IO ()
+  -> ExceptT CliError IO ()
 checkByronGenesisDelegation (CertificateFile certF) magic issuer delegate = do
-  ecert <- canonicalDecodePretty <$> LB.readFile certF
+  ecert <- liftIO $ canonicalDecodePretty <$> LB.readFile certF
   case ecert of
-    Left e -> throwIO $ DlgCertificateDeserialisationFailed certF e
+    Left e -> left $ DlgCertificateDeserialisationFailed certF e
```

Comment:
I prefer `throwError` to `left` :slightly_smiling_face:

Reply:
I prefer `left` to `throwError`. The former suggests a connection to the `Left` of an `Either` whereas the later uses "throw' which (incorrectly) suggests a connection to exceptions. `